source/draw/Color.ooc

```diff
@@ -137,6 +142,31 @@ ColorYuv: cover {
 	operator != (other: ColorBgra) -> Bool { !this equals(other) }
 }
 
+ColorYuva: cover {
+	yuv: ColorYuv
+	alpha: UInt8
+	init: func@ (=yuv, =alpha)
+	init: func@ ~uint8 (y, u, v, a: UInt8) {
+		this yuv = ColorYuv new(y, u, v)
+		this alpha = a
+	}
+	init: func@ ~default { this init(0, 128, 128, 255) }
+	init: func@ ~int (y, u, v, a: Int) { this init(y as UInt8, u as UInt8, v as UInt8, a as UInt8) }
+	init: func@ ~float (y, u, v, a: Float) { this init(y*255.0f clamp(0.0f, 255.0f) as UInt8, u*255.0f clamp(0.0f, 255.0f) as UInt8, v*255.0f clamp(0.0f, 255.0f) as UInt8, a*255.0f clamp(0.0f, 255.0f) as UInt8) }
```

Comment:
The check for `*` was removed because magic couldn't tell a `Float*` from a `1.0f * 2.0f`, I think. It gave false positives.